data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -297,8 +320,25 @@
 
         [message]
             speaker=Dacyn
-            message= _ "Very well. The best of luck in your battle."
+            message= _ "Very well. The best of luck in your battle. Gweddry, we may be able to avoid the bulk of the trolls by going through the dwarves' base."
         [/message]
+
+        [remove_shroud]
+            side=1
+            x,y=8-30,2-10
+        [/remove_shroud]
+        [remove_shroud]
+            side=1
+            x,y=31-36,4-10
+        [/remove_shroud]
+        [remove_shroud]
+            side=1
+            x,y=37-38,4-5
+        [/remove_shroud]
```

Comment:
Remove the shroud before the "going through the dwarves' base" line, so the player can see the alternative path while Dacyn's line is displayed?